test/unit-tests/org/userTest.js

```diff
@@ -506,6 +508,65 @@ describe('Test user functions in Org Controller', () => {
     })
   })
 
+  context('Getting all users of an org', () => {
```

Comment:
Since the repository refactoring exists in this branch, these tests should not be written in a way where they require DB access.